mlflow/protos/service.proto

```diff
@@ -455,16 +455,29 @@ service MlflowService {
     };
   }
 
-  // Create a new trace within an experiment.
-  rpc createTrace(CreateTrace) returns (CreateTrace.Response) {
+  // Start a new trace within an experiment.
+  rpc startTrace(StartTrace) returns (StartTrace.Response) {
     option (rpc) = {
       endpoints: [{
         method: "POST",
         path: "/mlflow/traces"
         since { major: 2, minor: 0 },
```

Comment:
minor 12? or 13 🤔

Reply:
We leave them all as part of API version 2.0 :)